source/uri-options/tests/auth-options.json

```diff
@@ -15,6 +15,19 @@
         },
         "authSource": "$external"
       }
+    },
+    {
+      "description": "Valid auth options are parsed correctly (SCRAM-SHA-1)",
+      "uri": "mongodb://foo:bar@example.com/?authMechanism=SCRAM-SHA-1&authSource=authSourceDB",
+      "valid": true,
+      "warning": false,
+      "hosts": null,
+      "auth": null,
+      "options": {
+        "authMechanism": "SCRAM-SHA-1",
+        "authMechanismProperties": null,
```

Comment:
I don't think we want to put `authMechanismProperties` here even with the value of `null`; the existing tests all just leave off fields that explicitly specified.

Reply:
Good point. Fixed that.